packaging/README.md

```diff
@@ -11,9 +11,14 @@ The recommendation is to install package within a Python virtual environment. Se
 
 ### Run package Module
 
-To package the connector:
+To package the connector and sign it:
 ```
-(.venv) PS connector-plugin-sdk\packaging> python -m package.package [path_to_folder]
+(.venv) PS connector-plugin-sdk\packaging> python -m package.package [path_to_folder] -a [alias_name] -ks [keystore_file_path]
+```
+
+To package the connector only:
```

Comment:
Change to "To package the connector without signing" to be a bit more explicit in what it means. Right now someone might assume that without the flag we'd package something in addition to the connector.

Reply:
Done, thanks.